src/test/kotlin/uk/gov/dluhc/printapi/logging/CorrelationIdMdcIntegrationTest.kt

```diff
@@ -56,6 +44,12 @@ internal class CorrelationIdMdcIntegrationTest : IntegrationTest() {
     @Autowired
     private lateinit var correlationIdRestTemplateClientHttpRequestInterceptor: CorrelationIdRestTemplateClientHttpRequestInterceptor
 
+    @Autowired
+    private lateinit var testScheduledJob: TestScheduledJob
```

Comment:
One question, @NathanRussellValtech ,

We usually add the autowired properties to the `IntegrationTest` class to avoid Spring spinning up a new context and slow down our integration tests (I guess that was the reason behind it).

Should we not follow the same pattern here as well?

Reply:
Yes, an excellent question, and you are right.
In this case I went with putting them straight in the test class because 
a) we want to be able to lift this code, inc this test class, straight out into the other VCA apps, and I'm aiming to be able to do it without having to mess (too much) with their integration test setup
b) longer term (and tbh not too far in the future) we'll be looking at pulling this out entirely into its own shared library artefact ... at which point we'll probably not have an IntegrationTest superclass, and just annotate the test class here  with SpringBootTest

Does that make sense? Or have I waffled a bit? 🤣

Reply:
That definitely makes sense, @NathanRussellValtech. Thanks for the answer.